airflow/dags/ingestion.py

```diff
@@ -19,21 +21,19 @@
 
 
 # CDC Covid Deaths
-cdc_dataset_name = 'cdc_covid_deaths'
-cdc_covid_deaths_bq_payload = {'message': {'is_airflow_run': True,
-                                           'filename': 'cdc_deaths',
-                                           'gcs_bucket': Variable.get('GCS_LANDING_BUCKET'),
-                                           'id': 'CDC_COVID_DEATHS'}}
-cdc_covid_deaths_gcs_payload = copy.deepcopy(cdc_covid_deaths_bq_payload)
-cdc_covid_deaths_gcs_payload['message']['url'] = 'https://data.cdc.gov/api/views/k8wy-p9cg/rows.csv?accessType=DOWNLOAD'
-cdc_covid_deaths_gcs_operator = create_gcs_ingest_operator(
+cdc_covid_deaths_gcs_payload = util.generate_gcs_payload(
+    'cdc_deaths', 'CDC_COVID_DEATHS', _CDC_COVID_DEATHS_DOWNLOAD_URL)
+cdc_covid_deaths_gcs_operator = util.create_gcs_ingest_operator(
     'cdc_covid_deaths_to_gcs', cdc_covid_deaths_gcs_payload, data_ingestion_dag)
-cdc_covid_deaths_bq_payload['message']['dataset'] = cdc_dataset_name
-cdc_covid_deaths_bq_operator = create_bq_ingest_operator(
+cdc_covid_deaths_bq_payload = util.generate_bq_payload(
+    'cdc_deaths', 'CDC_COVID_DEATHS', _CDC_DATASET_NAME)
```

Comment:
Make this a constant as well. 

Sorry I know this is the worst type of comment... "since you are already here...". So feel free to ignore if you want and I can change in a separate PR.

Reply:
np, done! :)